npm/qsharp/ux/estimatesPanel.tsx

```diff
@@ -14,6 +14,7 @@ export function EstimatesPanel(props: {
   runNames: string[];
   calculating: boolean;
   onRowDeleted: (rowId: string) => void;
+  allowSaveImage?: boolean;
```

Comment:
Here we have a `?` on `allowSaveImage`. Should this be added to `allowSaveImage` in the other files?

Reply:
I think this is alright - I like optional properties when there's more than one caller (so that we don't have to update all callers) , but with these components there's really only one caller, so there's not much point to it.